packages/common/src/operations.graphql

```diff
@@ -27,6 +27,7 @@ query invoice($id: String!) {
       shippedDatetime
       deliveredDatetime
       enteredByName
+      donorName
```

Comment:
Ah I just realised this was my addition to the type, but the generated code was melting down